### DISCUSSION TITLE
Connect to Kafka Cluster using Topic Operator without importing the existing topics

### DISCUSSION THREAD

**Discussion by amansbs** [strimzi/strimzi-kafka-operator#6625] [Q&A] [answered]
Hi, I am connecting my existing kafka cluster in AWS using the Topic Operator to create topics, As soon as I create the topic operator resources using the Yaml manifests, The topic operator imports all the existing topics as well(which I think is the expected behaviour). Is there any way in which the topic operator doesn't imports all the existing topics and only manages the ones that we create using the topic operator itself. Or can I restrict the topic operator that it cannot delete the topics and only creates those.
Thanks in Advance

**Comment by scholzj** [comment, marked as answer]
No, this is not possible. The operator requires that to understand what the current and desired states of the topics are.
